package.json

```diff
@@ -65,6 +65,7 @@
     "@types/semver": "^7.3.4",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
     "@typescript-eslint/parser": "^4.0.0",
+    "@types/webpack": "^4.41.26",
```

Comment:
Why include `@types/webpack`? That's webpack 4 - don't we want to remove this?

Reply:
Hmm, I had a hard time locating the types in webpack, so wasn't sure if they included types for everything yet. I'll take another look on Monday. I also assumed that most of the types for 4.4x.x will be the same for Webpack 5. Most things hasn't changed. Have a nice weekend!

Reply:
Yeah, either way we should not be depending on `@types/webpack` as it will clash with the types shipped with webpack 5. Have a nice weekend!

Reply:
Totally agree! I jumped to some conclusions there. Good catch!

Reply:
@johnnyreilly: Reverted those changes:
webpack.Loader.LoaderContext doesn't seem to exist.
There seems to be limited typings for Loaders. They use `any` internally. 
Do you want to add custom typings or not use them at all?

![image](https://user-images.githubusercontent.com/5993909/107114824-21a13580-6869-11eb-87aa-6e733e04475a.png)
![image](https://user-images.githubusercontent.com/5993909/107114835-3b427d00-6869-11eb-9392-ac1495dedb37.png)

Reply:
There's a missing piece. Somewhere out there is a doc that detailed the changes to webpacks APIs between 4 and 5. We need to track it down